www-cronrobot/app/views/layouts/application.html.erb

```diff
@@ -1,20 +1,92 @@
 <!DOCTYPE html>
-<html>
-  <head>
-    <title>CronRobot</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-  </head>
-
-  <body>
-    <% if @current_user %>
-      <%= button_to 'Logout', '/auth/logout', method: :get %>
-    <% else %>
-      <%= button_to 'Login', '/auth/auth0', method: :post %>
-    <% end %>
-    <%= yield %>
-  </body>
-</html>
+<html lang="en" dir="ltr">
+
+<head>
+  <!-- The header of the website, will contain title, description and meta tags with 3D images -->
+  <meta charset="utf-8">
+  <title>CronRobot: </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+  <meta name="robots" content="index, follow" />
+  <meta name="apple-mobile-web-app-capable" content="yes" />
+  <meta name="apple-touch-fullscreen" content="yes" />
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
+  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+  <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+</head>
+
+<body>
+
+  <!-- The responsive sidebar of the website, will only appear in mobile devices -->
+  <div id="mySidebar" class="sidebar">
+    <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+    <div id="sidebar-links">
+      <a onclick="handleLogin()">Dashboard</a>
```

Comment:
I am not sure it's clear to a new user that link.

If the user is not logged in:

- show login, signup links
- If he is logged in, then that link (Dashboard) makes sense

Reply:
ah alright sure, can be done easily

Reply:
![image](https://user-images.githubusercontent.com/61944452/130656069-ef57dbb4-114e-4d66-bfe6-2300b64fad82.png)
Done 👍